Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -154,12 +154,32 @@ def tests
       # Let `bundle` in PATH find its gem.
       ENV["GEM_PATH"] = "#{ENV["GEM_PATH"]}:#{gem_user_dir}"
 
+      # Submit test flakiness information using BuildPulse
+      use_buildpulse = ENV["HOMEBREW_BUILDPULSE_ACCESS_KEY_ID"].present? &&
+                       ENV["HOMEBREW_BUILDPULSE_SECRET_ACCESS_KEY"].present? &&
+                       ENV["HOMEBREW_BUILDPULSE_ACCOUNT_ID"].present? &&
+                       ENV["HOMEBREW_BUILDPULSE_REPOSITORY_ID"].present?
+      ENV["BUILDPULSE"] = "1" if use_buildpulse
```

Comment:
Where do we use this env variable?

Reply:
Ah, wait, I see it now. `HOMEBREW_BUILDPULSE`, maybe?

Reply:
@carlocab specifically not using `HOMEBREW` prefix to 1) be more like `CI` and 2) because they aren't filtered by `brew` when we're using `bundle exec` like this.

Reply:
Ok, that's fine then.